queries/appSettings.ts

```diff
@@ -87,12 +80,12 @@ export const getAnonymousRecruitmentStatus = createCachedFunction(async () => {
     'allowAnonymousRecruitment',
   );
   return getAnonymousRecruitmentStatus;
-}, ['allowAnonymousRecruitment', 'appSettings']);
+}, ['appSettings-allowAnonymousRecruitment', 'appSettings']);
```

Comment:
Sorry for not explaining this well - I was actually suggesting we should _remove_ these specific actions/queries and just use the generic get/set settings functions, since those can automatically set cache keys and call revalidate.